mx.py

```diff
@@ -101,9 +101,13 @@ def cpu_count():
     if is_jython():
         from java.lang import Runtime
         runtime = Runtime.getRuntime()
-        return runtime.availableProcessors()
+        cpus = runtime.availableProcessors()
     else:
-        return multiprocessing.cpu_count()
+        cpus = multiprocessing.cpu_count()
+    if hasattr(_opts, 'cpu_count'):
```

Comment:
Why are you using `hasattr` instead of `_opts.cpu_count`? Is this method called before option parsing has happened?